packages/vscode-extension/src/webview/hooks/useResizableProps.tsx

```diff
@@ -31,10 +35,17 @@ export function useResizableProps({
   const calculatePhoneDimensions = useCallback(
     (delta = 0) => {
       if (zoomLevel === "Fit") {
-        setPhoneHeight("100%");
-        setMaxWidth("100%");
+        const windowHeight = document.getElementById("root")?.clientHeight ?? 0;
```

Comment:
I think it would be nice to use media query in css here, aside from code aesthetic that  I think would improve, if you aggressively resize the hight in the current implementation you can see a little "jump" back up in size to the `MIN_FIT_ZOOM_LEVEL` when the device changes sizes between `(windowHeight < FIT_MODE_WINDOW_HEIGHT_THRESHOLD)` condition and other scenarios and I  think it could be avoided with media query.

Reply:
I don't think that a media query is a solution here unless we want to enforce MIN_FIT_ZOOM_LEVEL regardless of zoom mode. If yes then in that case media query is not necessary.